database-couchbase/config.js

```diff
@@ -0,0 +1,4 @@
+const cluster = new couchbase.Cluster("couchbase://localhost", {
+  username: "Administrator",
+  password: "password",
+});
```

Comment:
Just make sure to gitignore this. And maybe create an example one in case 'other people' will be working on it.